example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -348,15 +347,9 @@ describe('When there are granule differences and granule reconciliation is run',
     // need to add the cmr granule back to the table, so the granule can be removed from api
     await granuleModel.create(cmrGranule);
     await granulesApiTestUtils.removeFromCMR({ prefix: config.stackName, granuleId: cmrGranule.granuleId });
-    await waitForConceptExistsOutcome(cmrGranule.cmrLink, false);
```

Comment:
I'm sure it should be obvious, but why was this removed?

Reply:
That's a question for @nemreid , I'm not sure.

Reply:
Looks like it was pulled here https://github.com/nasa/cumulus/commit/c679e499955c7fc10c24e0a79f425042221ff2ab#diff-75c457041fb29817c2af29ba14ad2b4e

Reply:
Is this just a matter of master not being pulled into the TF branch?